src/database/models/index.js

```diff
@@ -10,6 +10,7 @@ logs.belongsTo(car, {
   targetKey: 'id',
 });
 
+logs.hook('afterCreate', log => car.update({ last_log_km: log.end_km }, { where: { id: log.car_id } }));
```

Comment:
cooool

Reply:
I think I know what this does, can you clarify tho

Reply:
As I said before https://github.com/FACK1/KamSafer/issues/43#issuecomment-464460700 :+1: